namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-data-compliance-preprod/resources/variables.tf

```diff
@@ -0,0 +1,42 @@
+variable "domain" {
+  default = "prison-data-compliance-preprod.prison.service.justice.gov.uk"
```

Comment:
I understand you are not creating a zone in this PR, but as you already got a zone with `prison.service.justice.gov.uk`, so you don't need this defined here.